lib/transactions.js

```diff
@@ -216,7 +216,45 @@ exports.createGeneration = (rpcData, blockReward, feeReward, recipients, poolAdd
                 Math.round(blockReward.total * (1 - (feePercent / 100))) + feeReward
             )
         }
-    } else {
+    
+    } else if(rpcData.coinbase_required_outputs && rpcData.coinbase_required_outputs.length){
```

Comment:
Can you fix the alignment in this block? (should be four spaces)

Reply:
Fixed.